api/include/opentelemetry/context/context_value.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <cstdint>
+
+#include "opentelemetry/nostd/span.h"
+#include "opentelemetry/nostd/unique_ptr.h"
+#include "opentelemetry/nostd/variant.h"
+#include "opentelemetry/version.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace context
+{
+using ContextValue = nostd::variant<bool, int64_t, uint64_t, double>;
```

Comment:
I think we'll need `nostd::shared_ptr<SpanContext>` too.

Reply:
When I tried to add it I got a: variant is implicitly deleted because the default definition would be ill-formed error

Reply:
So the nostd::shared_ptr<SpanContext> does work. But whenever I try to add a unique_ptr of any type, I am getting the implicitly deleted because of ill formed definition error.

Reply:
nostd::shared_pr<SpanContext> has been added is the unique_ptr necessary?